Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -66,12 +66,9 @@ struct ScheduleTransferForeignExchangeData: ScheduleTransferSectionData {
                 let destinationAmount = foreignExchange.destinationAmount,
                 let destinationCurrency = foreignExchange.destinationCurrency,
                 let rate = foreignExchange.rate {
-                let sourceAmountFormatted = String(format: "%@ %@",
-                                                   sourceAmount.format(with: sourceCurrency),
-                                                   sourceCurrency)
-                let destinationAmountFormatted = String(format: "%@ %@",
-                                                        destinationAmount.format(with: destinationCurrency),
-                                                        destinationCurrency)
+                let sourceAmountFormatted = sourceAmount.format(with: sourceCurrency)
+                let destinationAmountFormatted = destinationAmount.format(with: destinationCurrency)
```

Comment:
Arun no need to format the sourceAmount. So update it to `let destinationAmountFormatted = String(format: "%@ %@", destinationAmount, destinationCurrency)`